app/platform/templates/platform/developers/choose_question_type.html

```diff
@@ -0,0 +1,39 @@
+{% extends "platform/manage_grant_base.html" %}
+
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+{% from "govuk_frontend_jinja/components/radios/macro.html" import govukRadios %}
+
+
+{% block pageTitle %}
+    Add a question - {{ form.title }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_sub_navigation_tab = "grant_developers" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        "text": "Back",
+        "href": url_for("platform.manage_form", grant_id = grant.id, collection_id = collection.id, section_id=section.id, form_id=db_form.id, back_link="manage_section")
+    }) }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">What is the type of the question?</h1>
+            <form method="post" novalidate>
+
+                {{ form.csrf_token }}
+                {{ form.form_id }}
+
+                {{ form.question_data_type }}
+
+                {{ form.submit(params={
+                "text": "Continue"
+            }) }}
```

Comment:
`What is the type of the question?` should be the label on the form field, and then for simple one-question pages like this, use the isPageHeading params when rendering the question. This will mean the question is properly linked up to the input box and there's no duplication/redundancy of text on the page.

```suggestion
            <form method="post" novalidate>

                {{ form.csrf_token }}
                {{ form.form_id }}

                {{ form.question_data_type (params={"label": {"isPageHeading": true, "classes": "govuk-label--l" } } ) }}

                {{ form.submit(params={
                "text": "Continue"
            }) }}
```